kitten-scientists.user.js

```diff
@@ -209,7 +230,8 @@ var run = function() {
                     entangler:    {require: 'antimatter',  enabled: false},
 
                     // Centaurus
-                    tectonic: {require: 'antimatter', enabled: false}
+                    tectonic:      {require: 'antimatter', enabled: false},
+                    moltenCore:    {require: 'uranium',     enabled: false}
```

Comment:
Please keep spacing consistent with the rest of the file to keep code looking clean.